mne/tests/test_epochs.py

```diff
@@ -67,23 +67,28 @@ def test_handle_event_repeated():
                        [3, 0, 2], [3, 0, 1],
                        [5, 0, 2], [5, 0, 1], [5, 0, 3],
                        [7, 0, 1]])
-
+    selection = np.arange(3)
+    drop_log = [[]] * 3 + [['MEG 2443']] * 4
     with pytest.raises(RuntimeError, match='Event time samples were not uniq'):
-        _handle_event_repeated(EVENTS, EVENT_ID, event_repeated='error')
+        _handle_event_repeated(EVENTS, EVENT_ID, event_repeated='error',
+                               selection, drop_log)
```

Comment:
[Azure](https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=3605&view=logs&jobId=2e1ef983-081d-5c63-b72a-d9f329364ee0) says:
```
E     File "D:\a\1\s\mne\tests\test_epochs.py", line 74
E       selection, drop_log)
E       ^
E   SyntaxError: positional argument follows keyword argument
```

Reply:
The tests_epochs has problem but Alex said, he will take over. The original bug is fixed but the tests_epochs needs to be updated.